client/me/security-ssh-key/security-ssh-key.tsx

```diff
@@ -99,6 +101,7 @@ export const SecuritySSHKey = () => {
 	} );
 
 	const hasKeys = data && data.length > 0;
+	const redirectToHosting = queryArgs.source && queryArgs.siteSlug;
```

Comment:
shouldn't we be checking that the `source` is actually `hosting-config` rather than just that it is present?